integration/src/main/java/scripts/client/RawlsClient.java

```diff
@@ -1,13 +1,78 @@
 package scripts.client;
 
+import bio.terra.rawls.api.BillingV2Api;
+import bio.terra.rawls.api.WorkspacesApi;
 import bio.terra.rawls.client.ApiClient;
+import bio.terra.rawls.client.ApiException;
+import bio.terra.rawls.model.CreateRawlsV2BillingProjectFullRequest;
+import bio.terra.rawls.model.WorkspaceDetails;
+import bio.terra.rawls.model.WorkspaceRequest;
 import bio.terra.testrunner.common.utils.AuthenticationUtils;
 import bio.terra.testrunner.runner.config.ServerSpecification;
 import bio.terra.testrunner.runner.config.TestUserSpecification;
 import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.stream.Stream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RawlsClient {
+  private static final Logger log = LoggerFactory.getLogger(RawlsClient.class);
+
+  private static final String BILLING_ACCOUNT = "billingAccounts/00708C-45D19D-27AAFA";
```

Comment:
Should we make this configurable with an `integration.billingAccount` property or similar?